src/client/src/app/core/clock.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { DateRange } from '@features/charts/charts.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ClockService {
+  public today() {
+    return new Date(new Date().toDateString());
+  }
+
+  public todayRange() {
+    const today = this.today();
+    return {
+      dateFrom: today,
+      dateTo: today
+    } as DateRange;
+  }
+
+  public addDays(date: Date, count = 1) {
+    const newDate = new Date(date.valueOf());
+    newDate.setDate(newDate.getDate() + count);
+    return newDate;
+  }
+
+  public toUtcString(date: Date) {
+    return new Date(date.getTime() - date.getTimezoneOffset() * 60_000).toISOString();
```

Comment:
Just a comment, toISOString() returns only UTC time, if you know about that then it's okay :D

Reply:
the name of the function is `toUtcString` so that's the whole point ;P